src/main/webapp/app/exercises/text/assess/textblock-feedback-editor/textblock-feedback-editor.component.html

```diff
@@ -4,6 +4,16 @@
         <span [jhiTranslate]="'artemisApp.AssessmentType.' + feedback.type"></span>
     </span>
 
+    <span
+        class="badge bg-primary findOrigin"
+        *ngIf="feedback.type === FeedbackType.AUTOMATIC"
+        (click)="openOriginOfFeedbackModal(toggleMore)"
+        [ngbTooltip]="'artemisApp.textAssessment.feedbackEditor.automaticFeedbackInformationModal.trace' | artemisTranslate"
+        placement="top"
+    >
+        <fa-icon [icon]="'search'"></fa-icon>
```

Comment:
I think a text would make sense here, intuitively I would not expect the badge to be clickable - especially since the other one is not. Something like "Trace Origin". There is surely enough space left for that.

Reply:
agreed. search icon with Trace Feedback text would be better.

Reply:
that's a pretty good point already 👍 . the only thing is, we are planning to add some more icons adjacent to this 'Automatic' label in an upcoming PR (something like upvote/downvote, etc). Our idea is we don't want to saturate it a lot with lots of labels, and keep it plain simple. 

The most important one is the 'Automatic' label, so in any case that would be the longest text - the main message being sent, with the icons being secondary helpers only. they do and will have a label though on :hover, I would find it that over time, you'd hover at least once. Especially if there are many icons, that would make it obvious that sth is going on there.

Reply:
ohh I see that makes sense when you add the other icons...